cylc/flow/scripts/cylc_stop.py

```diff
@@ -128,32 +127,19 @@ def main(parser, options, suite, shutdown_arg=None):
             pclient, "suite stopped", options.interval, options.max_polls)
 
     if options.wall_clock:
-        prompt(
-            'Set shutdown at wall clock %s for %s' % (
-                options.wall_clock, suite),
-            options.force)
```

Comment:
@hjoliver maybe this is causing the issue in the functional test?

If you go to the GitHub actions page, then click on download raw logs, and open one of the logs for the running functional tests, you should find something like this:

```
2020-08-13T17:49:06.8394683Z [17:49:06] tests/f/execution-time-limit/02-slurm.t ........................... skipped: "[test battery][batch systems][slurm]host" not defined
2020-08-13T17:49:07.2539755Z No matching tasks found: c.1
2020-08-13T17:49:07.2540057Z 
2020-08-13T17:49:08.9355240Z [17:49:08] tests/f/jinja2/11-logging.t ....................................... ok     3510 ms ( 0.00 usr  0.01 sys +  4.65 cusr  0.40 csys =  5.06 CPU)
2020-08-13T17:49:09.2717669Z Usage: cylc [control] stop|shutdown [OPTIONS] REG [STOP] 
2020-08-13T17:49:09.2718185Z 
2020-08-13T17:49:09.2718441Z Tell a suite server program to shut down. In order to prevent failures going
2020-08-13T17:49:09.2718679Z unnoticed, suites only shut down automatically at a final cycle point if no
2020-08-13T17:49:09.2718912Z failed tasks are present. There are several shutdown methods:
2020-08-13T17:49:09.2719101Z 
2020-08-13T17:49:09.2719307Z   1. (default) stop after current active tasks finish
2020-08-13T17:49:09.2720023Z   2. (--now) stop immediately, orphaning current active tasks
2020-08-13T17:49:09.2720414Z   3. (--kill) stop after killing current active tasks
2020-08-13T17:49:09.2720653Z   4. (with STOP as a cycle point) stop after cycle point STOP
2020-08-13T17:49:09.2720877Z   5. (with STOP as a task ID) stop after task ID STOP has succeeded
2020-08-13T17:49:09.2721251Z   6. (--wall-clock=T) stop after time T (an ISO 8601 date-time format e.g.
2020-08-13T17:49:09.2721612Z      CCYYMMDDThh:mm, CCYY-MM-DDThh, etc).
2020-08-13T17:49:09.2721818Z 
2020-08-13T17:49:09.2722027Z Tasks that become ready after the shutdown is ordered will be submitted
2020-08-13T17:49:09.2722273Z immediately if the suite is restarted.  Remaining task event handlers and job
2020-08-13T17:49:09.2722506Z poll and kill commands, however, will be executed prior to shutdown, unless
2020-08-13T17:49:09.2722828Z --now is used.
2020-08-13T17:49:09.2723017Z 
2020-08-13T17:49:09.2723367Z This command exits immediately unless --max-polls is greater than zero, in
2020-08-13T17:49:09.2723610Z which case it polls to wait for suite shutdown.
2020-08-13T17:49:09.2723910Z 
2020-08-13T17:49:09.2724119Z Arguments:
2020-08-13T17:49:09.2724334Z    REG                  Suite name
2020-08-13T17:49:09.2724554Z    [STOP]               a/ task POINT (cycle point), or
2020-08-13T17:49:09.2724931Z                             b/ ISO 8601 date-time (clock time), or
2020-08-13T17:49:09.2725178Z                             c/ TASK (task ID).
2020-08-13T17:49:09.2725344Z 
2020-08-13T17:49:09.2725676Z cylc-shutdown: error: no such option: -f
2020-08-13T23:41:02.1313259Z ##[error]The operation was canceled.
2020-08-13T23:41:02.1364824Z Post job cleanup.
2020-08-13T23:41:02.2116090Z [command]/usr/bin/git version
2020-08-13T23:41:02.2173504Z git version 2.28.0
2020-08-13T23:41:02.2199599Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2020-08-13T23:41:02.2229513Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2020-08-13T23:41:02.2494242Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2020-08-13T23:41:02.2521237Z http.https://github.com/.extraheader
2020-08-13T23:41:02.2528282Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2020-08-13T23:41:02.2560645Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2020-08-13T23:41:02.2816913Z Cleaning up orphan processes
2020-08-13T23:41:02.2951295Z Terminate orphan process: pid (5263) (prove)
2020-08-13T23:41:02.2981635Z Terminate orphan process: pid (81310) (bash)
2020-08-13T23:41:02.2994668Z Terminate orphan process: pid (89092) (cylc-run)
2020-08-13T23:41:02.3012886Z Terminate orphan process: pid (91285) (sleep)
```

Looks like a `shutdown -f` was executed in one of the tests?

Reply:
Ah, good spotting :+1:

Reply:
(Will leave this for you to deal with @MetRonnie ...)